backend/services/syson-services/src/main/java/org/eclipse/syson/services/DiagramDirectEditListener.java

```diff
@@ -216,6 +216,8 @@ public void exitAbstractPrefixExpression(AbstractPrefixExpressionContext ctx) {
                 usage.setIsAbstract(true);
             } else {
                 usage.setIsAbstract(false);
+                // A variation is always abstract,
```

Comment:
Please remove final `,`, here and everywhere else in this PR

Reply:
Done